cdap-docs/tracker-manual/source/index.rst

```diff
@@ -133,8 +147,8 @@ To enable Tracker, go to the **Tracker UI** at
 (Distributed CDAP) http://host:dashboard-bind-port/ns/default/tracker/home 
 and press the ``"Enable Tracker"`` button to deploy and start Tracker.
```

Comment:
in standalone it is automatically enabled. in distributed, you have to manually enable it.